### PR TITLE
Extend walletconsole tests

### PR DESCRIPTION
**Pull request by catenocrypt**
## Description

Extend walletconsole tests for coverage.

## Testing instructions

Unit tests, coverage report.

## Types of changes

* Minor refactor, extra tests.

## Checklist

- [x] Prefix PR title with `[WIP]` if necessary.
- [x] Add tests to cover changes as needed.
- [x] Update documentation as needed.

### CODE REVIEW

File: walletconsole/lib/WalletConsole.cpp [hunk 1]
```diff
@@ -13,9 +13,12 @@
 #include "TonCoin.h"
 
 #include "Base64.h"
+//#include "Base58.h"
 #include "HexCoding.h"
 #include "Data.h"
 
+#include "FIO/TransactionBuilder.h" // TODO
```
Comment (hewigovens) [inline walletconsole/lib/WalletConsole.cpp]: Is this finished?

Comment (catenocrypt) [reply walletconsole/lib/WalletConsole.cpp]: Ops, I checked in the wrong file.  Corrected.

**Comment by catenocrypt** [comment]
Coverage build is red :(  I think due to the large change in the baseline (AnySigner), but coverage is increased :)

**Comment by catenocrypt** [comment]
Coverage build is red :(  I think due to the large change in the baseline (AnySigner), but coverage is increased :)

**Pull request merged**